dist/debian/Dockerfile

```diff
@@ -45,8 +45,8 @@ RUN \
 				~# Copy over template if HOME empty\r\
 				[ "$(ls -A $HOME)" ] || cp -R /tmp/home_template/. $HOME\r\
 				\# Source shell intercept\r~g' | tr -d '\n' | tr -d '\t')" \
-			-e "$(echo 's~\t# Install Conda dependencies\r\
-				~\t# Post start hook\r\
+			-e "$(echo 's~\t# Install Conda dependencies\r~\
```

Comment:
Are we sure that this change to the regex properly adds the hook? Might need to confirm that the `add_entrypoint` function is still added.

Reply:
`add_entrypoint` function is still added :+1: